packages/core/src/bootstrap.ts

```diff
@@ -65,7 +66,21 @@ export async function bootstrap(userConfig: Partial<VendureConfig>): Promise<INe
         tokenMethod === 'cookie' || (Array.isArray(tokenMethod) && tokenMethod.includes('cookie'));
     if (usingCookie) {
         const { cookieOptions } = config.authOptions;
-        app.use(cookieSession(cookieOptions));
+        app.use(
```

Comment:
Can we wrap this whole part (from `const { tokenMethod } = ...` in a function `configureSessionCookies()`?

Then it can be exported and used inside the test server too to avoid the repetition.

Reply:
Done in [`203fd90` (#2482)](https://github.com/vendure-ecommerce/vendure/pull/2482/commits/203fd900c3db382b76dc8ef8efa7ed1efe05afc8) 👍